api/v1alpha1/clusterconfig_types.go

```diff
@@ -15,7 +15,9 @@ import (
 )
 
 const (
-	CNIProviderCalico = "calico"
+	CNIProviderCalico                 = "calico"
+	CSIProviderAWSEBS                 = "aws-ebs-sc"
```

Comment:
What does "sc" mean here, maybe just this?
```suggestion
	CSIProviderAWSEBS                 = "aws-ebs"
```

Reply:
storage class